### ISSUE TITLE
[Accessibility]: extension Publish button

### ISSUE THREAD

**Issue by abeatrix** [sourcegraph/sourcegraph#34839]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34050

### Problem description

After clicking on the Publish button, the page would reload, but the focus would be removed so it is not clear if the action was performed or not:
![image](https://user-images.githubusercontent.com/68532117/166500126-9c179bed-ec44-44df-851c-a3d0193772d7.png)

### Expected behavior

After clicking on the button it would let me know what the button has done, in this case, if the extension is published

### Additional details

Add `aria-live="polite"` to the button div so that when the page reloads it would read the status line next to the button.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
